concepticondata/conceptlists/Heeschen-1978-104.tsv

```diff
@@ -0,0 +1,106 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Heeschen-1978-1	1	all	98	ALL
+Heeschen-1978-2	2	ashes	646	ASH
+Heeschen-1978-3	3	bark	1204	BARK
+Heeschen-1978-4	4	belly	1251	BELLY
+Heeschen-1978-5	5	big	1202	BIG
+Heeschen-1978-6	6	bird	937	BIRD
+Heeschen-1978-7	7	let us bite	1403	BITE
+Heeschen-1978-8	8	black	163	BLACK
+Heeschen-1978-9	9	blood	946	BLOOD
+Heeschen-1978-10	10	bone	1394	BONE
+Heeschen-1978-11	11	breast	1402	BREAST
+Heeschen-1978-12	12	let us burn	2102	BURN
```

Comment:
since this is transitive, it should be [burn (something)](http://concepticon.clld.org/parameters/141) rather than the general "burn".